src/Patterns.php

```diff
@@ -1,7 +1,5 @@
 <?php namespace LeanNs;
```

Comment:
Yes this is done on the following patterns since this just address the removal of `Timber`